static/src/javascripts/projects/common/modules/preferences/main.js

```diff
@@ -73,11 +75,127 @@ define([
                     });
 
                 React.renderComponent(React.createElement(SummaryTagsSettings), placeholder);
+            },
+
+            initialiseNotificationPreferences = function () {
+                var isPushEnabled = false;
+
+                var pushButton = document.querySelector('.js-push-button');
```

Comment:
the class is a bit misleading (most buttons get pushed). `.js-push-notifications-button`